### PR TITLE
17/local storage

### PR DESCRIPTION
**Pull request by j-dav**
- Quick implementation of localStorage
- Renders the form using the retrieved settings

### CODE REVIEW

File: src/components/form/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+const Form = () => {
+	const [settings, setSettings] = useState();
+
+	useEffect(() => {
+		const data = fetchLocalStorage();
+		setSettings(data);
+	}, []);
+
+	const handleSubmit = (e) => {
```
Comment (j-mes) [inline src/components/form/index.js:16]: can we use `error` instead of `e` ? Makes it more readable and maintainable?

Comment (j-dav) [reply src/components/form/index.js:16]: e is actually `event`, i'll change it to avoid that confusion

Comment (j-mes) [reply src/components/form/index.js:16]: 😄 intentional btw ;)

File: src/components/form/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { h } from 'preact';
```
Comment (j-mes) [inline src/components/form/index.js:1]: Is this really a form component or should it be named something else? 🤔

File: src/lib/storage-helper.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { settingsObj } from '../config';
```
Comment (j-mes) [inline src/lib/storage-helper.js:1]: Can we change from `./lib/<thing>-helper.js` to `./helpers/local-storage.js`? Makes it more intuitive.

**Review by j-mes — COMMENTED** [review]
Couple comments, looks good though!

File: src/components/form/index.js (hunk 2 above)
Comment (j-dav) [reply src/components/form/index.js:1]: Its just a placeholder name for now, I have not done any styling or anything. It was purely to test the local storage with a form. Our components will be created later that way we can name it something suitable as I imagine the inputs, labels etc will become their own components then we can construct forms wherever in the same style.

File: src/lib/storage-helper.js (hunk 3 above)
Comment (j-dav) [reply src/lib/storage-helper.js:1]: fair point

File: src/components/form/index.js (hunk 2 above)
Comment (j-dav) [reply src/components/form/index.js:1]: When we start making the components we won't see much code, it will be injected via props etc and mapped

Comment (j-mes) [reply src/components/form/index.js:1]: Hmm I'd suggest putting a

```// TODO: Revise the component name```

comment in there if it helps? As it'll remind us to revisit this at a later date

**Review by j-mes — APPROVED** [review]
Happy to go with this once you've added the `// TODO` comment block please @j-dav

**Pull request merged**